R/summary_rows.R

```diff
@@ -122,7 +122,11 @@ summary_rows <- function(data,
 
   # Derive the summary labels
   summary_labels <-
-    vapply(fns, derive_summary_label, FUN.VALUE = character(1))
+    fns %>%
+    vapply(
+      FUN.VALUE = character(1),
+      FUN = derive_summary_label
+    )
```

Comment:
Let's not go out of our way to pipe things, they make it a little more annoying to debug